src/Shared/@components/Input/Input.module.scss

```diff
@@ -0,0 +1,33 @@
+.root {
+  display: flex;
```

Comment:
And why do you need flex? What wasn't working with block?

Reply:
Nothing, senpai! To be fixed, senpai!